### DISCUSSION TITLE
I am using vc-measure-area component  some times it is crashing

### DISCUSSION THREAD

**Discussion by sachinsbgithub** [zouyaoji/vue-cesium#365] [Q&A]
I am using vc-measure-area component  some times it is crashing (showing  0sqm
![Screenshot 2023-03-14 104437](https://user-images.githubusercontent.com/114548996/224906455-51bf3432-b7c2-4a51-b21d-8b3b4518a31d.jpeg)
![Screenshot 2023-03-14 104512](https://user-images.githubusercontent.com/114548996/224906461-5b9398bf-7d55-439b-9561-fe0c5e0267ac.jpeg)

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium-v2/#/zh/tool/vc-measuring

Online documentation, can't reproduce it, is your Cesium the official version?
